documentation/docs/analytics.md

```diff
@@ -0,0 +1,94 @@
+---
+sidebar_position: 8
+---
+
+# Analytics
+The MonkJs SDK includes built-in support for capturing and managing analytics events such as user interactions, page views, and custom events.
+Internally, we use [PostHog](https://posthog.com) to manage our analytics, and you are free to use our PostHog infrastructure
+or use your preferred analytics tool.
+
+![npm latest package](https://img.shields.io/npm/v/@monkvision/analytics/latest.svg)
+
+## Why use Analytics?
+While you can opt out of our analytics features when using our SDK, we highly recommend utilizing them for the following reasons:
+
+- **Enhanced Insights**: Gain valuable insights into user interactions and behaviors in your app by leveraging our ready-to-use analytics providers.
+- **Real-Time Tracking**: Monitor user behavior and application usage patterns in real-time, allowing for proactive improvements.
+- **Support Eligibility**: Monk does not offer tech support for deployed applications and projects where analytics are disabled.
+
+
+## Enabling Analytics
+To enable analytics support in your application, wrap your application in an `AnalyticsProvider` component,
+exported by the `@monkvision/analytics`, and pass it the Analytics Adapter(see the Analytics Adapter section below):
```

Comment:
...and pass it **an** Analytics Adapter...

also add a space after _Analytics Adapter_